md_core_helps/one_time/wikiblame/bot.py

```
@@ -58,21 +61,21 @@ def fetch_content(self) -> None:
         url = f"{self.base_url}?{urlencode(self.params)}"
         # ---
         if "printurl" in sys.argv:
-            print(url)
+            logger.info(url)
         # ---
         response = requests.get(url)
```

Ruff (0.14.14)

[error] Probable use of `requests` call without timeout

(S113)